### DISCUSSION TITLE
error while using the @timestamp field as a http_post2_payload body

### DISCUSSION THREAD

**Discussion by Sinistr0** [jertel/elastalert2#1106] [Q&A] [answered]
hello, I'm having an error when I try to use the @timestamp field as the body of the payload, I believe it's because of the @ at the beginning of the word, is there any way around this?
I'm trying to use this way " `timestamp: {{@timestamp}}` "

**Comment by jertel** [comment, marked as answer]
If you upgrade to the latest release, 2.10.0, you can access it via `_data['@timestamp']`. See the [docs](https://elastalert2.readthedocs.io/en/latest/ruletypes.html#http-post-2) for more information.
